### PR TITLE
[KED-3046] Remove references to quantum black labs where appropriate

### PR DESCRIPTION
**Pull request by SajidAlamQB**
## Description

There's a few mentions that kedro is authored by QuantumBlack Labs (e.g. in kedro docs, setup.,py, `kedro info`). Following the migration to LF these shouldn't be there any more.

Best way to find them is just to search "quantumblack" in the codebase - there's not that many results to check through.

Additionally you will also have to update URLs for Kedro-Airflow, Kedro-Docker and Kedro-Telemetry as they have completely changed repo location.

As well as kedro repository, you should check kedro-community, kedro-plugins, kedro-viz and kedro-training. kedro-starters has already been updated.

https://jira.quantumblack.com/browse/KED-3046

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: kedro-airflow/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Kedro-Airflow
-[![CircleCI](https://circleci.com/gh/quantumblacklabs/kedro-airflow/tree/master.svg?style=shield)](https://circleci.com/gh/quantumblacklabs/kedro-airflow/tree/master)
+[![CircleCI](https://circleci.com/gh/kedro-org/kedro-plugins/tree/main.svg?style=shield)](https://circleci.com/gh/kedro-org/kedro-plugins/tree/main)
```
Comment (MerelTheisenQB) [inline kedro-airflow/README.md:2]: I'm wondering if it makes sense to keep this badge here if it's not specific to the plugin anymore. (same for the other readme.md files)

**Review by MerelTheisenQB — APPROVED** [review]
Looks good! 👍

File: kedro-airflow/README.md (hunk 1 above)
Comment (SajidAlamQB) [reply kedro-airflow/README.md:2]: Yeh, I agree feels a bit odd having it here now unless there is a way to make it specific to each plugin again?

Comment (AntonyMilneQB) [reply kedro-airflow/README.md:2]: I don't think there's any way to make it specific to a plugin unfortunately. We can point the badge to a particular branch but not a particular CircleCI workflow. So I think just remove it.

**Review by AntonyMilneQB — APPROVED** [review]
Good stuff! Just a small point about the CircleCI badges 🌟

**Pull request merged**